src/test/scala/is/hail/io/ImportVCFSuite.scala

```diff
@@ -206,11 +206,15 @@ class ImportVCFSuite extends SparkSuite {
   @Test def notIdenticalHeaders() {
     val tmp1 = tmpDir.createTempFile("sample1", ".vcf")
     val tmp2 = tmpDir.createTempFile("sample2", ".vcf")
+    val tmp3 = tmpDir.createTempFile("sample3", ".vcf")
 
     val vds = hc.importVCF("src/test/resources/sample.vcf")
     val sampleIds = vds.sampleIds
     vds.filterSamplesList(Set(sampleIds(0))).exportVCF(tmp1)
     vds.filterSamplesList(Set(sampleIds(1))).exportVCF(tmp2)
-    intercept[SparkException] (hc.importVCFs(Array(tmp1, tmp2)))
+    assert(intercept[SparkException] (hc.importVCFs(Array(tmp1, tmp2))).getMessage.contains("invalid sample ids"))
+
+    vds.filterSamplesList(Set(sampleIds(0),sampleIds(1))).exportVCF(tmp3)
```

Comment:
There's a function `TestUtils.interceptFatal` that will do this. Example usage:

```
TestUtils.interceptFatal("Duplicate"){ kt2.annotate("X = group(foo, a, b, b)") }
```

Reply:
Good to know, thanks! So in this case `TestUtils.interceptFatal("invalid sample ids"){ hc.importVCFs(Array(tmp1, tmp2)) }` would do the same thing?

Reply:
I _believe_ that the interceptFatal uses regex, so in this case it would be the same, but not always. We should expand it to do either regex (Default) or plaintext matching, that would be nice!